great_expectations/core/util.py

```diff
@@ -733,31 +733,34 @@ class DBFSPath:
     Methods for converting Databricks Filesystem (DBFS) paths
     """
 
+    @staticmethod
```

Comment:
I reversed the order of these methods in order to better match the order in which they are used (the first typically for Pandas; the second -- for Spark).